_posts/2024-08-25-mot-tracking-metrics.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title: "Understanding MOT Metrics"
+subtitle: "Comparing MOTA, IDF1, and HOTA for Multi-Object Tracking Evaluation"
+description: "Explore the essential metrics in Multi-Object Tracking (MOT) with this comprehensive guide. We break down and compare key metrics like MOTA, IDF1, and HOTA, while also covering DetA and AssA. Understand how these metrics assess spatial accuracy and temporal consistency in object tracking."
+image: "/assets/images/fullsize/posts/2024-08-25-mot-tracking-metrics/thumbnail.jpg"
+selected: y
+mathjax: y
+---
+
+After a long time, I have finally sat down to write this blog post on tracking metrics. It builds on my last post about [tracking by detection]({% post_url 2023-11-08-tracking-by-detection-overview %}) and delves deeper into how we measure tracking performance.
+
+In this article, I'll provide an introduction to tracking metrics, starting from the basic principles and breaking down the key differences between various metrics. I'll focus on three popular metrics: MOTA, IDF1, and HOTA, which are widely used in the Multi-Object Tracking (MOT) community. Understanding these is crucial because the choice of metric can significantly impact how we interpret a tracker's performance.
+
+Let's get started!
+
+## The basics
+
+**DetA** (Detection Accuracy) measures how well a tracker localizes objects in each frame, typically using Intersection over Union (IoU) thresholds. It essentially quantifies the spatial accuracy of detections. 
+
+<div class="post-center-image">
+    {% picture pimage /assets/images/fullsize/posts/2024-08-25-mot-tracking-metrics/iou.png --alt IoU diagram %}
+</div>
+
+{:refdef: class="image-caption"}
+*Figure 1. IoU diagram [from jonathanluiten](https://jonathonluiten.medium.com/how-to-evaluate-tracking-with-the-hota-metrics-754036d183e1){:target="_blank"}{:rel="noopener noreferrer"}*
+{: refdef}
+
+**AssA** (Association Accuracy), on the other hand, evaluates how accurately a tracker maintains object identities across frames. It focuses on the temporal consistency of ID assignments, measuring how well the tracker links detections of the same object over time. See, for example, the image below, extracted from the HOTA paper, where different tracking associations (A, B, C) are shown for the ground truth track (GT).
```

Comment:
1. Add a reference to the HOTA paper when you cite it.
2. I miss a definition of what exactly this metric measures. You just say it measures the accuracy, but how?